src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -4701,16 +4701,13 @@
      * <p>parses as:</p>
      *
      * <pre>
-     * +--PACKAGE_DEF (package)
-     *     |
-     *     +--ANNOTATIONS
-     *         |
-     *         +--ANNOTATION
-     *             |
-     *             +--AT (&#064;)
-     *             +--IDENT (MyAnnotation)
-     *     +--IDENT (blah)
-     *     +--SEMI (;)
+     * `--PACKAGE_DEF -> package
+     *     |--ANNOTATIONS -> ANNOTATIONS
```

Comment:
Please remove leading '`--', and indent whole block to left

Reply:
I meant

From: ``--PACKAGE_DEF`
To `PACKAGE_DEF`

Reply:
oh okay